README.md

```diff
@@ -104,7 +104,7 @@ var kernel = Kernel.Builder.Build();
 
 // For Azure Open AI details please see
 // https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api
-kernel.Config.AddAzureOpenAITextCompletion(
+kernel.Config.AddAzureOpenAITextCompletionService(
```

Comment:
nit: we shouldn't update this yet until nuget is spun

Reply:
Looks like we spun out a NuGEt.